code/modules/mapping/mapping_helpers.dm

```diff
@@ -245,6 +245,139 @@
 	else
 		airlock.autoname = TRUE
 
+//apc helpers
+/obj/effect/mapping_helpers/apc
+	desc = "You shouldn't see this. Report it please."
+	layer = DOOR_HELPER_LAYER
+	late = TRUE
+
+/obj/effect/mapping_helpers/apc/Initialize(mapload)
+	. = ..()
+	if(!mapload)
+		log_mapping("[src] spawned outside of mapload!")
+		return
+
+	var/obj/machinery/power/apc/target = locate(/obj/machinery/power/apc) in loc
+	if(isnull(target))
+		log_mapping("[src] failed to find an apc at [AREACOORD(src)]")
+	else
+		payload(target)
+
```

Comment:
```suggestion
		payload(target)
	
	return INITIALIZE_HINT_LATELOAD

```

Functionally we don't need this since we preserve the return type from parent with the `. = ..()` but I think it is much better to be safe than sorry- we really want these to continue to work and ensure that LateInitialize is called in any needed situation.